spotfire/spk.py

```diff
@@ -838,7 +838,7 @@ def _handle_versioning(package_builder, installed_packages, brand, version, forc
         package_builder.version = given_version
     # Handle versioned filenames
     if versioned_filename:
-        package_builder.output = re.sub(r"(\.spk)?$", r"-%s\1" % str(package_builder.version),
+        package_builder.output = re.sub(r"(\.spk)?$", f"-{str(package_builder.version)}\1",
```

Comment:
Minor point: you can combine the `f` and `r` string modifiers.  In this case it actually has an impact on the regexp backreference (changing it to an ASCII 1 character):

```python
>>> r"\1"
'\\1'
>>> f"\1"
'\x01'
>>> fr"\1"
'\\1'
```

Reply:
Good catch!  Updated to use "fr" string instead of just "f" string.